lib/flop.ex

```diff
@@ -108,6 +122,23 @@ defmodule Flop do
     embeds_many :filters, Filter
   end
 
+  @nil_meta %Meta{
```

Comment:
I think it would be more reasonable to remove `@enforce_keys` from the meta module instead.